src/QuartzNET-DynamoDB/DataModel/Storage/Repository.cs

```diff
@@ -4,16 +4,26 @@
 using System.Net;
 using Amazon.DynamoDBv2;
 using Amazon.DynamoDBv2.Model;
+using Polly;
 
 namespace Quartz.DynamoDB.DataModel.Storage
 {
-    public class Repository<T> : IRepository<T>, IDisposable where T : IInitialisableFromDynamoRecord, IConvertibleToDynamoRecord, IDynamoTableType, new()
+    public class Repository<T> : IRepository<T> where T : IInitialisableFromDynamoRecord, IConvertibleToDynamoRecord, IDynamoTableType, new()
     {
-        private AmazonDynamoDBClient _client;
+        private readonly AmazonDynamoDBClient _client;
+        private readonly Policy _writeRetryPolicy;
+        private readonly Policy _readRetryPolicy;
 
         public Repository(AmazonDynamoDBClient client)
         {
             _client = client;
+            _writeRetryPolicy = Policy
+                .Handle<ProvisionedThroughputExceededException>()
+                .WaitAndRetry(5, ExponentialBackoffWithRandomVariation.CalculateWaitDuration);
+
+            _readRetryPolicy = Policy
+             .Handle<ProvisionedThroughputExceededException>()
+             .WaitAndRetry(3, ExponentialBackoffWithRandomVariation.CalculateWaitDuration);
```

Comment:
i'm happy with it as is, can look into it if this ever appears to be problematic.